internal/scheduler/jobdb.go

```diff
@@ -111,33 +316,37 @@ func (it *JobQueueIterator) NextJobItem() *SchedulerJob {
 	return jobItem
 }
 
+// Next is needed to implement the memdb.ResultIterator interface.  External callers should use NextJobItem which
+// provides a typesafe mechanism for getting the next SchedulerJob
 func (it *JobQueueIterator) Next() interface{} {
 	return it.NextJobItem()
 }
 
+// jobDbSchema() creates the database schema.
+// This is a simple schema consisting of a single "jobs" table with indexes for fast lookups
 func jobDbSchema() *memdb.DBSchema {
 	indexes := make(map[string]*memdb.IndexSchema)
-	indexes["id"] = &memdb.IndexSchema{
-		Name:    "id",
+	indexes[idIndex] = &memdb.IndexSchema{
```

Comment:
so the name of the index is still `id` it's just been broken out into a const variable called `idIndex`.  We could rename the variable but I'm mildly opposed because although it's obvious from the context here, it's not obvious in other places where we need to use it.